src/models/Scholarships.test.ts

```diff
@@ -19,6 +19,7 @@ function create(data: {
   amount?: ScholarshipAmount | undefined;
   name?: string;
   deadline?: Date;
+  grades?: number[];
```

Comment:
```suggestion
  grades?: GradeLevel[];
```

You're going to need to use the `GradeLevel` types elsewhere in tests (you can unwrap them to avoid the `GradeLevel.` prefix if you'd like).

Reply:
what do you mean by unwrapping them?

Reply:
I guess in JS they call it [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment). I.e. if you can't do
```js
import { MIDDLE_SCHOOL, ... } from '../types/GradeLevel.ts'
```
You could do this at the top of the tests:
```js
const { MIDDLE_SCHOOL, ... } = GradeLevel;
```
So that you don't need to write the `GradeLevel.` prefix everywhere